src/test/java/uk/gov/pay/api/it/fixtures/PaymentResultBuilder.java

```diff
@@ -174,10 +162,11 @@ protected TestPaymentFailureState(String status, String message, String code) {
     protected String returnUrl;
     protected String paymentProvider;
     protected String createdDate;
-    protected List<ImmutableMap<?, ?>> links = new ArrayList<>();
+    protected List<Map<?, ?>> links = new ArrayList<>();
     protected RefundSummary refundSummary;
     protected SettlementSummary settlementSummary;
     protected String gatewayTransactionId;
+    protected Map<String, ?> metadata;
```

Comment:
Having noticed this in `CardPayment` should this use `ExternalMetadata` too, I think there could be an exception here since its a test fixture which ultimately just needs a Map to serialize to JSON.